packages/taquito-rpc/src/taquito-rpc.ts

```diff
@@ -779,4 +780,41 @@ export class RpcClient {
     return this.url
   }
 
+  /**
+   *
+   * @param options contains generic configuration for rpc calls
+   *
+   * @description Voting period of current block.
+   *
+   * @see https://tezos.gitlab.io/008/rpc.html#get-block-id-votes-current-period
```

Comment:
@kjaiswal 

Can you update the URLs to use `api` in place of the proto number? ie `https://tezos.gitlab.io/api/rpc.html#get-block-id-votes-current-period` 

The API website should always redirect the API to the latest protocol, so it should mean less maintenance for us.

Reply:
Can you add `@example` props too?

Adding an `@example` prop for the typical case (head) and for a specific block is very helpful to developers.

Reply:
Thank you for the feedback I have added @example props and replaced 008 with api.